src/Fmt_ast.ml

```diff
@@ -3585,7 +3601,8 @@ and fmt_structure_item c ~last:last_item ?ext {ctx; ast= si} =
                ?ext:(if first then ext else None)
                ctx binding
                ?epi:
-                 (Option.some_if c.conf.sparse
+                 (Option.some_if
+                    (break_cases_level c > 0)
```

Comment:
This ident  does not really make sense anymore. This has nothing to do with  break_cases.

Reply:
Ah yes, this should probably be its own option, perhaps `extra-line-after-multiline-binding` or something. I'm actually tempted to remove this extra space entirely and see if anyone complains.